### ISSUE TITLE
[Bug]: Clicking folder names for navigation shows black border for a split second

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/server#41490]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

When I click on a folder with my mouse

https://github.com/nextcloud/server/assets/1374172/42b54fd0-0a54-4361-819d-08bf7fd9a4f6

### Steps to reproduce

1. Create a folder
2. Click on its name with your mouse

### Expected behavior

Black border only shows if I navigate by keyboard

### Installation method

None

### Nextcloud Server version

28

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by ShGKme** [comment]
This is a focus state of a link

**Comment by ChristophWurst** [comment]
It's still happening with 28RC3. Backport?
